app/presenters/two_factor_authentication/selection_presenter.rb

```diff
@@ -36,17 +36,33 @@ def info
     def mfa_configuration_count; end
 
     def mfa_configuration_description
-      return '' if !disabled?
-      t(
-        'two_factor_authentication.two_factor_choice_options.configurations_added',
-        count: mfa_configuration_count,
-      )
+      return '' if mfa_configuration_count === 0
```

Comment:
Typically we want to avoid the triple equal in Ruby for simple comparisons like this, since it doesn't meant the same as in other languages (e.g. JavaScript) as being "strictly" equal.

https://medium.com/rubyinside/triple-equals-black-magic-d934936a6379

```suggestion
      return '' if mfa_configuration_count == 0
```

Also, I'm wondering about the change from `.zero?` to this comparison. Was that to make it `nil`-tolerant? But then we also updated the implementations of `mfa_configuration_count` above to always return `0` and not `nil`, so are we expecting `nil` values from that method or no?

Reply:
Fixed that triple equals.
I actually changed it because the test was failing reporting a .zero? as an undeclared method or something like that. It was working in the app but breaking the test.

Reply:
I think my concern is that it seems like `mfa_configuration_count` can sometimes return `nil`, which is why you were seeing errors. So if we're expecting it to return `mfa_configuration_count`, we should handle that in the logic and have test coverage for it.